google/earthenterprise
# Increase performance logging

Adding additional logging at the NFY_PROGRESS level to more easily analyze very large and slow builds.  Each message will identify the location in the method of interest and indicate the elapsed time from the previous performance log.

Note: I'm unsure if this is desirable to merge to master or if it should only live in the experimental branch.

Verification steps:

1. Build the branch for this PR and install on target
1. Change logging level to 3 or higher in `/etc/opt/google/systemrc`
1. Restart or reload gefusion service
1. Perform a build of an imagery or terrain project.
1. View the additional log output and ensure it all makes sense and is usable.
